<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QsoDataForm</class>
 <widget class="QGroupBox" name="QsoDataForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QSO Data Form</string>
  </property>
  <property name="title">
   <string>QSO Data Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,2,0,1,0,1">
     <item row="0" column="5">
      <widget class="QComboBox" name="qsoBandCb">
       <property name="toolTip">
        <string>QSO band</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDateTimeEdit" name="qsoDateTimeOff">
       <property name="minimumSize">
        <size>
         <width>162</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>QSO end date/time (UTC)</string>
       </property>
       <property name="displayFormat">
        <string>MM/dd/yyyy hh:mm</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Callsign:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="qsoCallsignTxt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Contact callsign</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QComboBox" name="qsoModeCb">
       <property name="toolTip">
        <string>QSO mode</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDateTimeEdit" name="qsoDateTimeOn">
       <property name="minimumSize">
        <size>
         <width>162</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>QSO start date/time (UTC)</string>
       </property>
       <property name="displayFormat">
        <string>MM/dd/yyyy hh:mm</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mode:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Band:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="qsoRstSentTxt">
       <property name="toolTip">
        <string>Readability/Signal Strength/Tone sent</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Freq:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>RST Sent:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Time On:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TimeOff:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>QTH:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>RST Recv:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="qsoFreqMhzTxt">
       <property name="toolTip">
        <string>QSO frequency (MHz)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="qsoRstRecvTxt">
       <property name="toolTip">
        <string>Readability/Signal Strength/Tone received</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Power:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLineEdit" name="qsoPowerWattsTxt">
       <property name="toolTip">
        <string>Tx power (Watts)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="qsoQthTxt"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0,1,0,2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Country:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="qsoCountryCb"/>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>State:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="qsoPriAdminCb"/>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>County:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="qsoSecAdminCb"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Comments:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>QSO Message:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="qsoCommentsTxt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="toolTip">
        <string>QSO notes and comments</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="qsoMessageTxt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Message for contact</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qsoCallsignTxt</tabstop>
  <tabstop>qsoDateTimeOn</tabstop>
  <tabstop>qsoBandCb</tabstop>
  <tabstop>qsoModeCb</tabstop>
  <tabstop>qsoDateTimeOff</tabstop>
  <tabstop>qsoRstSentTxt</tabstop>
  <tabstop>qsoRstRecvTxt</tabstop>
  <tabstop>qsoFreqMhzTxt</tabstop>
  <tabstop>qsoPowerWattsTxt</tabstop>
  <tabstop>qsoQthTxt</tabstop>
  <tabstop>qsoCountryCb</tabstop>
  <tabstop>qsoPriAdminCb</tabstop>
  <tabstop>qsoSecAdminCb</tabstop>
  <tabstop>qsoCommentsTxt</tabstop>
  <tabstop>qsoMessageTxt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
